.github/CODEOWNERS

```diff
@@ -132,7 +132,7 @@
 
 # ServiceLabel: %Communication - SMS
 # PRLabel: %Communication - SMS
-/sdk/communication/azure-communication-sms/                           @DimaKolomiiets @gfeitosa-msft @besh2014 @phermanov-msft @ilyapaliakou-msft
+/sdk/communication/azure-communication-sms/                           @gfeitosa-msft @besh2014 @phermanov-msft @ilyapaliakou-msft @dinazavyr
```

Comment:
Maybe while we are already editing this, let's also add Ilkin and Kirill T.

Reply:
I will ad them separately later because until they respond until they check they memebership in Azure. It might take time